packages/frontend/core/src/modules/quicksearch/impls/links.ts

```diff
@@ -31,11 +31,29 @@ export class LinksQuickSearchSession
   query$ = new LiveData('');
 
   items$ = LiveData.computed(get => {
-    const query = get(this.query$);
-    if (!query) return [];
+    const query = get(this.query$).trim();
+    const result: QuickSearchItem<'link', LinkPayload>[] = [];
+    if (!query) return result;
 
-    const isLink = query.startsWith('http://') || query.startsWith('https://');
-    if (!isLink) return [];
+    let isLink = false;
+
+    try {
+      let url = query;
+      const hasScheme = ['http://', 'https://'].some(s => url.startsWith(s));
+
+      if (!hasScheme) {
+        const dotIdx = url.indexOf('.');
+        if (dotIdx > 0 && dotIdx < url.length - 1) {
```

Comment:
```suggestion
        if (dotIdx !== -1) {
```

Reply:
When dot is at the end, it is not recognized as domian